### PR TITLE
Tablet progress bar

### PR DESCRIPTION
**Pull request by RikJurriaans**
Add breakpoints and logic for breakpoints

### CODE REVIEW

File: src/styles/package.json [hunk 1]
```diff
@@ -12,7 +12,9 @@
     "build": "npm run clean && tsc"
   },
   "dependencies": {
-    "@emotion/core": "^10.0.16"
+    "@emotion/core": "^10.0.16",
+    "@types/facepaint": "^1.2.1",
```
Comment (brendonboshell) [inline src/styles/package.json]: Small thing: if this is just types, it should probably be in `devDependencies`.

Comment (RikJurriaans) [reply src/styles/package.json]: ow, cool, i will do that

File: src/styles/package.json [hunk 2]
```diff
@@ -12,9 +12,11 @@
     "build": "npm run clean && tsc"
   },
   "dependencies": {
-    "@emotion/core": "^10.0.16"
+    "@emotion/core": "^10.0.16",
+    "facepaint": "^1.2.1"
   },
   "devDependencies": {
     "typescript": "^3.6.2"
+    "@types/facepaint": "^1.2.1"
```
Comment (brendonboshell) [inline src/styles/package.json]: Missing comma here :)

Comment (meraymond2) [inline src/styles/package.json]: comma?

**Pull request merged**